pkg/client/events_query/client_integration_test.go

```diff
@@ -0,0 +1,55 @@
+//go:build integration
+
+package eventsquery_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"pocket/internal/testclient/testeventsquery"
+)
+
+func TestQueryClient_EventsObservable_Integration(t *testing.T) {
```

Comment:
Stupid question, but as I'm looking at this test, it's not clear to me where the publishing of NewBlock is coming from.

Can you #PUC a bit so its clearer?